packages/effect/src/SchemaAST.ts

```diff
@@ -2075,6 +2075,13 @@ export const getTemplateLiteralCapturingRegExp = (ast: TemplateLiteral): RegExp
   return new RegExp(pattern)
 }
 
+/**
+ * @since 3.?.?
+ */
+export const getLiteralCapturingRegExp = (ast: Literal): RegExp => {
+  return new RegExp(`^(${regexp.escape(String(ast.literal))})$`)
```

Comment:
`ast.literal` is a broad type, not just `string`; wasn't sure what to do with that